AsyncDisplayKit/ASTableView.mm

```diff
@@ -455,6 +455,14 @@ - (void)reloadRowsAtIndexPaths:(NSArray *)indexPaths withRowAnimation:(UITableVi
   [_dataController reloadRowsAtIndexPaths:indexPaths withAnimationOptions:animation];
 }
 
+- (void)relayoutRowAtIndexPath:(NSIndexPath *)indexPath withRowAnimation:(UITableViewRowAnimation)animation
+{
+  ASDisplayNodeAssertMainThread();
+  ASCellNode *node = [self nodeForRowAtIndexPath:indexPath];
+  [node setNeedsLayout];
```

Comment:
I wonder if we should also call -layoutIfNeeded here?  Mainly thinking that most users will want to do animated expansions of cells, etc...if we force layout here then they could capture the relayout's actions in an animation block placed around the call to this method.  Perhaps there are other side effects of that technique that suggest we should avoid it?  Unless we can think of any, let's change it to go that route.  Sometime soon we should create an example app with an animated row or item height change & internal relayout.